eos/rare-b-decays/nonlocal-formfactors-p-to-p.cc

```diff
@@ -610,6 +635,14 @@ namespace eos
         return abs(this->_imp->nff->Hhat_plus(q2));
     }
 
+    template <typename Process_>
+    double
+    NonlocalFormFactorObservable<Process_, nff::PToP>::abs_normalized_plus(const double & q2) const
+    {
+        return abs(this->_imp->nff->normalized_plus(q2));
```

Comment:
``abs`` could be ``::abs`` (i.e., the C version) rather than ``std::abs``. Can you make sure to add ``using std::abs`` somewhere at the top of the ``.cc`` file, but within the ``eos`` namespace?